web/src/components/auth/Confirmation.tsx

```diff
@@ -0,0 +1,98 @@
+import React, { useState } from 'react';
+import TextField from "@material-ui/core/TextField";
+import { styled } from "@material-ui/core/styles";
+import { Auth } from "aws-amplify";
+import Button from "@material-ui/core/Button";
+import CircularProgress from "@material-ui/core/CircularProgress";
+import { Link, useHistory } from "react-router-dom";
+import { Grid } from '@material-ui/core';
+import RightOnLogo from "./RightOnLogo.png";
+
+const Confirmation: React.FC = () => {
+  const [loading, setLoading] = React.useState(false);
+  const [email, setEmail] = useState("");
+  const [code, setCode] = useState("");
+  const history = useHistory();
+
+  const handleSubmit = async (e: React.SyntheticEvent<Element, Event>) => {
+    e.preventDefault();
+    setLoading(true);
+
+    try {
+      await Auth.confirmSignUp(email, String(code));
+      alert("Success");
```

Comment:
Same for this and the below alert. Generally, for errors, we might wanna show some message in the UI to tell the user "username or password was wrong" or something like that. Maybe run it by the UX team for what copy to be used